location-service/src/main/java/com/movie/recommender/location/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.movie.recommender.location.config;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.*;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Slf4j
+@Configuration
+public class RabbitMQConfig {
```

Comment:
this config is identical to the one used in the `crawler`
move it to a `common` project to avoid duplication and enable reuse